### ISSUE TITLE
Deps analyzer warnings when enabling files_external

### ISSUE THREAD

**Issue by PVince81** [owncloud/core#18434]
### Steps

1. Setup OC master
1. Enable "External Storage" app
1. Check logs

### Expected
No warnings

### Actual
```
{"reqId":"ipGxpA+QFd8xqqbs3LEf","remoteAddr":"127.0.0.1","app":"PHP","message":"Argument 1 passed to OC\\App\\DependencyAnalyzer::analyze() must be of the type array, string given, called in \/srv\/www\/htdocs\/owncloud\/lib\/private\/app.php on line 1142 and defined at \/srv\/www\/htdocs\/owncloud\/lib\/private\/app\/dependencyanalyzer.php#51","level":3,"time":"2015-08-19T16:09:34+00:00","method":"POST","url":"\/owncloud\/index.php\/settings\/ajax\/enableapp.php"}
{"reqId":"ipGxpA+QFd8xqqbs3LEf","remoteAddr":"127.0.0.1","app":"PHP","message":"Argument 2 passed to OC\\App\\DependencyAnalyzer::analyzeOC() must be of the type array, string given, called in \/srv\/www\/htdocs\/owncloud\/lib\/private\/app\/dependencyanalyzer.php on line 65 and defined at \/srv\/www\/htdocs\/owncloud\/lib\/private\/app\/dependencyanalyzer.php#279","level":3,"time":"2015-08-19T16:09:34+00:00","method":"POST","url":"\/owncloud\/index.php\/settings\/ajax\/enableapp.php"}
```

The warnings seem harmless and don't affect the functionality.

### Versions
ownCloud 8.2 git master (5fd36d017ecdbce61dcf1c67ae04c8c0be81a4bf)

@LukasReschke @DeepDiver1975 @Raydiation

**Comment by Xenopathic** [comment]
cc @nickvergessen

**Comment by MorrisJobke** [comment]
Fix is in #18439
